ts/features/wallet/component/__test__/PagoPaPaymentCapability.test.tsx

```diff
@@ -124,7 +124,7 @@ describe("PagoPaPaymentCapability", () => {
     );
     expect(component.getByText("Incompatible")).not.toBeNull();
   });
-  it('should render a badge with test "Incompatible" if passed a privative card, payment method of kind CreditCard with issuerAbiCode and type = PRV', () => {
+  it(' **DEPRECATED** should render a badge with test "Incompatible" if passed a privative card, payment method of kind CreditCard with issuerAbiCode and type = PRV', () => {
```

Comment:
Why? If this test is not necessary we should remove it. What do you think?

Reply:
adressed in https://github.com/pagopa/io-app/pull/4615/commits/b9230048e2ad74c789fb95c5efbdf376e86a5186